internal/oidcserver/types.go

```diff
@@ -13,6 +13,7 @@ type OIDCServer interface {
 	IssueToken(w http.ResponseWriter, r *http.Request)
 	IssueCertificate(w http.ResponseWriter, r *http.Request)
 	UserInfo(w http.ResponseWriter, r *http.Request)
+	Healthz(w http.ResponseWriter, r *http.Request)
```

Comment:
This is not needed.

Reply:
removed and moved to private function here  https://github.com/aporeto-inc/oidc-mock/commit/44ad77460f3b18e28a26ad561b2ff233f992add1